app/server/appsmith-server/src/main/java/com/appsmith/server/configurations/SecurityConfig.java

```diff
@@ -85,6 +91,12 @@ public class SecurityConfig {
     @Autowired
     private RedirectHelper redirectHelper;
 
+    @Value("${appsmith.internal.username}")
+    private String INTERNAL_USER_NAME;
```

Comment:
Minor, but this'll be uniform with the env variable.

```suggestion
    private String INTERNAL_USERNAME;
```